.github/workflows/build-test.yml

```
@@ -0,0 +1,116 @@
+          npx cypress run
+
+  lighthouse:
+    name: Lighthouse audit
+    needs: deploy-preview
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Run Lighthouse
+        uses: treosh/lighthouse-ci-action@v10
```

actionlint (1.7.7)

the runner of "treosh/lighthouse-ci-action@v10" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)